internal/trace/transport.go

```diff
@@ -34,6 +37,9 @@ var (
 	}
 )
 
+// payloadSizeLimit limits the maximum size of the response body to be printed.
+const payloadSizeLimit int64 = 4 * 1024 // 4 KiB
```

Comment:
@FeynmanZhou Please review the size limit. 4KiB can be easily reached but 4MiB is definitely too large.

@Wwwsylvia Can you test against real registries with 8KiB and 16KiB?

Reply:
The question is, what's the idea of limiting the body size? If it's to avoid the log to scroll too much, we can set a small number like 16KiB. If it's to avoid blow up the memory, 4 MiB makes sense as it's the normal manifest size limit.
FYI, the catalog response body against MAR is around 94 KiB.

Reply:
Let's take 16 KiB.